clonelists/Nintendo - GameCube (Redump).json

```diff
@@ -938,6 +938,15 @@
 				{"searchTerm": "Naruto - Gekitou Ninja Taisen! 2"}
 			]
 		},
+		{
+			"group": "Naruto - Gekitou Ninja Taisen! 4",
+			"titles": [
+				{"searchTerm": "Naruto - Gekitou Ninja Taisen! 4"}
+			],
+			"supersets": [

```

Comment:
Is there a reason for this to be a superset?

If it doesn't contain extra content and you just want the Rev 1 to win, you can add it as another title and Retool figures out the rest:

```json
"titles": [
  {"searchTerm": "Naruto - Gekitou Ninja Taisen! 4"},
  {"searchTerm": "Naruto - Gekitou Ninja Taisen! 4 (Taikai Sen'you Disc)"}
],
```